src/utils.ts

```diff
@@ -367,6 +370,18 @@ const utils = {
     return (type === 'function' || type === 'object') && !!obj;
   },
 
+  // taken from https://github.com/feross/is-buffer/blob/master/index.js
+  isBuffer: (obj: unknown): boolean => {
```

Comment:
should we just do `instanceof Uint8Array` instead of calling this function?

Reply:
I did that for the `multipart`-related usages, but for webhooks we rely on `Buffer.toString`, which behaves differently from `Uint8Array.toString`. There seem to be use cases where users rely on this behavior and pass in a `Buffer`: https://github.com/vendure-ecommerce/vendure/blob/master/packages/payments-plugin/src/stripe/stripe.service.ts#L72

Reply:
Can we use `TextDecoder` to mimic the behavior?

Reply:
Higher order principle here is that non-node customer should be able to use all Stripe Node SDK APIs so we can't have things that work only when `Buffer` is available.

Reply:
Ah I didn't realize that we could use `TextDecoder` on `Buffer` as well, updated!